axelrod/tests/unit/test_player.py

```diff
@@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
 
     name = "Player"
```

Comment:
That's true re other strategy test but don't we want to test the specific
initialisation of the global Player class? That feels like it should be
here and in this PR (I realise you've done loads on this and sound busy so
I'm happy to take that on).

On Tue, Sep 8, 2015 at 2:05 PM Owen Campbell <notifications@github.com>
wrote:

> In axelrod/tests/unit/test_player.py
> <https://github.com/Axelrod-Python/Axelrod/pull/293#discussion_r38922297>:
>
> > @@ -20,7 +20,9 @@ class TestPlayerClass(unittest.TestCase):
> >
> >      name = "Player"
>
> We should - but that's one of the four additional tasks I described in the
> gitter channel.
>
> I think each strategy test needs a expected_behaviour dictionary with all
> dimensons specified.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Axelrod-Python/Axelrod/pull/293/files#r38922297>.
>